test/DetectorInfo/DetectorPropertiesStandard_test.cc

```diff
@@ -123,6 +123,8 @@ main(int argc, char const** argv)
                                << " kV/cm"
                                << "\nSampling rate:       " << TDCtick << " ns"
                                << "\nArgon temperature:   " << detp.Temperature() << " K"
+                               << "\nArgon density:       " << detp.Density() << " kg/dm^3 (at " << detp.Temperature() << " K)"
+                               << "\nArgon density:       " << detp.Density(83.0) << " kg/dm^3 (at 87 K)"
```

Comment:
Should this be `detp.Density(87.0)`?

Reply:
It should have.
I have tried editing on GitHub... let's see if I managed.